BookingsApi/BookingsApi.IntegrationTests/Database/Commands/AddJudiciaryPersonCommandTests.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Threading.Tasks;
+using BookingsApi.DAL;
+using BookingsApi.DAL.Commands;
+using BookingsApi.DAL.Queries;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace BookingsApi.IntegrationTests.Database.Commands
+{
+    public class AddJudiciaryPersonCommandTests : DatabaseTestsBase
+    {
+        private AddJudiciaryPersonCommandHandler _commandHandler;
+        private GetJudiciaryPersonByExternalRefIdQueryHandler _getJudiciaryPersonByExternalRefIdQueryHandler;
+
+        [SetUp]
+        public void Setup()
+        {
+            var context = new BookingsDbContext(BookingsDbContextOptions);
+            _commandHandler = new AddJudiciaryPersonCommandHandler(context);
+            _getJudiciaryPersonByExternalRefIdQueryHandler = new GetJudiciaryPersonByExternalRefIdQueryHandler(context);
+        }
+        
+        [Test]
+        public async Task should_add_person()
+        {
+            var externalRefId = Guid.NewGuid();
```

Comment:
don't forget to clean up after your tests

Reply:
Done.. TearDown in the base class